lib/v2/lemmy/index.js

```diff
@@ -0,0 +1,52 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const md = require('markdown-it')({ html: true });
+
+module.exports = async (ctx) => {
+    const sort = ctx.params.sort ?? 'Active';
+    const community = ctx.params.community;
+    const communitySlices = community.split('@');
+    if (communitySlices.length !== 2) {
+        throw new Error(`Invalid community: ${community}`);
+    }
+    const instance = community.split('@')[1];
+
+    const communityUrl = `https://${instance}/api/v3/community?name=${community}`;
+    const communityData = await ctx.cache.tryGet(communityUrl, async () => {
+        const result = await got({ method: 'get', url: communityUrl, headers: { 'Content-Type': 'application/json' } });
+        return result.data.community_view.community;
+    });
+
+    const postUrl = `https://${instance}/api/v3/post/list?type_=All&sort=${sort}&community_name=${community}&limit=50`;
+    const postData = await ctx.cache.tryGet(postUrl, async () => {
+        const result = await got({ method: 'get', url: postUrl, headers: { 'Content-Type': 'application/json' } });
+        return result.data;
+    });
```

Comment:
For listing cache, the [4th parameter of tryGet](https://docs.rsshub.app/joinus/advanced/use-cache#ctxcachetrygetkey-getvaluefunc--maxage--refresh-) should be `false`. Otherwise, the list won't receive any updates once it's cached.
The cache age should be `routeExpire`.